runtime/src/main/resources/messages.properties

```diff
@@ -14,5 +14,9 @@
 # limitations under the License.
 #
 
-io.syndesis.credential.RelativeUri.message = Given URI value must be relative
-io.syndesis.model.validation.UniqueProperty.message = Value `${validatedValue[value]}` for property `{value}` is not unique
+io.syndesis.credential.RelativeUri.message          = Given URI value must be relative
+io.syndesis.model.validation.UniqueProperty.message = Value is not unique
```

Comment:
Same here (would be could to see the actual non-unique value in the error message)